pram

```diff
@@ -149,6 +156,24 @@ main() {
 			-S|--no-signoff)
 				signoff=
 				;;
+			-p|--part-of)
+				partof=1
+				;;
+			--part-of-pr)
+				partof=1
+				[[ -z ${user_pr} ]] || die "${0}: cannot specify multiple ${1}"
+				[[ ${#} -gt 1 ]] || die "${0}: missing argument to ${1}"
+				user_pr=${2}
+				shift
+				;;
+			--part-of-pr=*)
+				partof=1
+				[[ -z ${user_pr} ]] || die "${0}: cannot specify multiple ${1%%=*}"
+				user_pr=${1#*=}
+				;;
+			-P|--no-part-of)
```

Comment:
Similarly, I wonder if this option shouldn't also disabling adding the implicit `Closes`.